deployments/cert_manager.go

```diff
@@ -260,6 +261,32 @@ func (cm CertManager) apply(ctx context.Context, c *kubernetes.Cluster, ui *term
 	// Epinio's private CA. Phase 1, the CA root certificate, signed by self
 	// signed.
 
+	// Create an empty secret that the cert manager will fill-in with values.
+	// We do that, because we want to put the "kubed.appscode.com/sync" annotation
+	// as per the docs:
+	// https://cert-manager.io/docs/faq/kubed/#syncing-arbitrary-secrets-across-namespaces-using-kubed
+	secretName := "epinio-ca-root"
+
+	emptySecret := v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      secretName,
+			Namespace: CertManagerDeploymentID,
+			Annotations: map[string]string{
+				"kubed.appscode.com/sync": fmt.Sprintf("kubed-source-namespace=%s", CertManagerDeploymentID),
```

Comment:
Ok, so this signals to me the source namespace of the secret, when I see it on some namespace.
That's good.
You could shorten this, as the kubed connection is already clear from the key name:
```suggestion
				"kubed.appscode.com/sync": fmt.Sprintf("source-namespace=%s", CertManagerDeploymentID),
```

Reply:
On the namespace resource, the key is simply `kubed-source-namespace` and `kubed.appscode.com/sync` doesn't exist.

So this annotation is saying "when you find a namespace with a label key `kubed-source-namespace` and a label value of `CertManagerDeploymentID`, copy this secret over"